docs/user/Kubeconfig.md

```diff
@@ -0,0 +1,52 @@
+# Kubernetes Configuration File Types
+
+## Overview
+
+### Notes - viya4-deployment:6.6.0
+
+The release of kubectl v1.26 is dropping support for built-in provider-specific code in their project for authentication and instead opting for a plugin-based strategy. A quote from the [Google blog](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke) provides an explanation for this change:
+
+>To ensure the separation between the open source version of Kubernetes and those versions that are customized by services providers like Google, the open source community is requiring that all provider-specific code that currently exists in the OSS code base be removed starting with v1.26.
+
+### Usage with provider based kubeconfig files from viya4-iac-gcp:4.5.0
```

Comment:
Again version info concern.

Reply:
See reply above https://github.com/sassoftware/viya4-deployment/pull/435#discussion_r1187861765 for rationale and options.